source/patterns/@aws-solutions-constructs/aws-lambda-secretsmanager/test/lambda-secretsmanager.test.ts

```diff
@@ -0,0 +1,387 @@
+/**
```

Comment:
Request to add a unit test case for overriding `readonly secretProps?: secretsmanager.SecretProps;` the most common use case I can think of is passing in CMK `SecretProps.encryptionKey` to be used instead of the default `aws/secretsmanager` key for secret encryption

Reply:
Updated thanks @hnishar !